tests/semanticdb/metac.expect

```diff
@@ -12,12 +12,12 @@ Occurrences => 19 entries
 Symbols:
 example/Access# => class Access
 example/Access#`<init>`(). => primary ctor <init>
-example/Access#m1(). => method m1
-example/Access#m2(). => method m2
-example/Access#m3(). => method m3
-example/Access#m4(). => method m4
-example/Access#m5(). => method m5
-example/Access#m6(). => method m6
+example/Access#m1(). => method private[this] m1
+example/Access#m2(). => method private[this] m2
+example/Access#m3(). => method protected[example/Access#] m3
```

Comment:
if we look in `tests/semanticdb/expect/Access.scala` this should be `private` qualified not `protected` qualified

Reply:
Fixed by https://github.com/lampepfl/dotty/pull/12964#discussion_r661940556